css/styles.css

```diff
@@ -0,0 +1,77 @@
+/* styles.css */
+.team {
+  border: 1px solid #ccc;
+  padding: 1rem;
+  text-align: center;
+  cursor: pointer;
+  background: white;
+  border-radius: 8px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+  transition:
+    transform 0.2s,
+    box-shadow 0.2s;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
```

Comment:
Do you need this to be a flex column too? You are using `div`s in here which are block and stack in columns too. Are you using it for spacing?